pkg/api/v1/routes/handlers.go

```diff
@@ -163,8 +163,71 @@ func (r *Routes) serverConditionCreate(c *gin.Context) (int, *v1types.ServerResp
 		}
 	}
 
+	return r.conditionCreate(otelCtx, newCondition, serverID, facilityCode)
+}
+
+func (r *Routes) serverEnroll(c *gin.Context) (int, *v1types.ServerResponse) {
+	otelCtx, span := otel.Tracer(pkgName).Start(c.Request.Context(), "Routes.serverEnroll")
+	span.SetAttributes(
+		attribute.KeyValue{Key: "serverId", Value: attribute.StringValue(c.Param("id"))},
+		attribute.KeyValue{Key: "IP", Value: attribute.StringValue(c.Param("ip"))},
+		attribute.KeyValue{Key: "user", Value: attribute.StringValue(c.Param("user"))})
+	defer span.End()
+
+	repo, ok := r.repository.(*store.Serverservice)
+	if !ok {
+		return http.StatusInternalServerError, &v1types.ServerResponse{
+			Message: "add-server only support server-service store",
+		}
+	}
+
+	serverID, err := uuid.Parse(c.Param("uuid"))
```

Comment:
You have two parameters referencing the same concept: `id` (line 172) and `uuid` (here). I suggest 2 things:

1. Make the parameter name `serverID` in the POST payload (that is, what comes through on the `*gin.Context`)
2. Handle this as an optional parameter. If it's missing, create one via `uuid.New()`